internal/controller/capacity.go

```diff
@@ -93,13 +93,21 @@ func (r *InstasliceReconciler) findNodeAndDeviceForASlice(ctx context.Context, i
 					v1.ResourceMemory: memoryRequest,
 				},
 			)
+			// update deployed pod total metrics
+			if err := r.UpdateDeployedPodTotalMetrics(string(allocResult.Nodename), gpuuuid, allocRequest.PodRef.Namespace, allocRequest.PodRef.Name, allocRequest.Profile, allocResult.MigPlacement.Size); err != nil {
```

Comment:
I recommend we update metrics outside this method.

Reply:
I am updating the metrics in that function as we discussed to ensure we are using the up to date instaslice object

Reply:
this call has nothing to do with `findNodeAndDeviceForASlice`, it should be called after this method exists.